awx/main/models/host_metrics.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
+
+class HostMetrics(models.Model):
```

Comment:
I'd put this in `awx.main.models.inventory` near the `class Host` definition since it's pretty similar.

Reply:
okay , i will change that.

Reply:
I have updated it